modular_skyrat/modules/wirebrush_rust/code/modules/food_and_drinks/kitchen_machinery/microwave.dm

```diff
@@ -0,0 +1,99 @@
+/obj/machinery/microwave/attackby(obj/item/O, mob/living/user, params)
```

Comment:
Use descriptive variable names

Reply:
this is just modularization, a copy paste of the core code.